front/src/Phaser/Login/SelectCharacterScene.ts

```diff
@@ -243,6 +247,11 @@ export class SelectCharacterScene extends AbstractCharacterScene {
     }
 
     update(time: number, delta: number): void {
+        this.pointerTimer -= delta;
```

Comment:
You should add a comment here explaining why we need to do this (why we need the workaround), otherwise, it is not obvious and we will not remember when looking at this code in 6 monthes